### ISSUE TITLE
Hail & Ride

### ISSUE THREAD

**Issue by duiveldoder** [osmandapp/OsmAnd#14055]
I'm going to write this report a bit longer - with a bit more details as a common "issue" as I feel this issue is severely neglected by OSMAnd yet crucial if we wish to gain support for OSM.

my personal interests: I want to promote OSM - and trying to - among schools here in my area in the Philippines. And one of the most convincing arguments I found is that "jeepney" lines are usually only known to locals - local as in really from local village or local Barangay - because even locals of one part of the city/village don't know anything about Jeep lines in the other part of the same village/city. 

The big IT giant - not going to use the name here - is completely in the dark about this either. So here's really a chance for OSM to be promoted. Which I'm trying! 

Here's a link where you can see the "Jeepney" lines I'm adding to the city (Work in Progress)
[https://www.openstreetmap.org/#map=14/16.4096/120.6007&layers=ON](url)

But - when I want to show how it works with the navigation in OSM - using public transportation - it works great what already is implemented but the one thing that's very common here is the "Hail and Ride" concept and it's absolutely missing in OSMAnd.

So instead of telling people:" you can hail and ride that part of the road" and using that for the navigator - it tells people to take a walk to any known "stop location" in the area.. 

Which then results - that even if you're standing on a road where there's "Hail and Ride" and all you gotta do is wait a few minutes - the app tells people to walk down 150m elevation and 1km distance in total of steps and footways and roads, sometimes slippery - where all people have to do - is just wait.

This is the approved Hail and Ride site on wiki: [https://wiki.openstreetmap.org/wiki/Proposed_features/hail_and_ride](url)

And it's a valid member now of the route: [https://wiki.openstreetmap.org/wiki/Relation:route](url)

And it's again referred to here on the general public transportation page: [https://wiki.openstreetmap.org/wiki/Public_transport](url)

![image](https://user-images.githubusercontent.com/15773876/162641445-cea1c6bd-6ed4-48b5-a9c6-e4afc69e59e7.png)
This image shows the "first route" out of a list of options OSMAnd comes up with. Note none of the options use the "Hail and Ride" feature of OSM.

Here it let's you walk from the place I'm at - down the steps which is in total 150m elevation (it's really steep!!!) to the main provincial highway to an official stop where several Jeeps pass by indeed. So nothing wrong with that - except that the first choice should be the "just wait for the Hail and Ride jeep" which passes right where you are right now.

![image](https://user-images.githubusercontent.com/15773876/162641542-5c14a515-80f2-44db-ba54-de31b3ad564a.png)
Now on this photo you see 3 of the 7 options OSM comes up with - all 7 are valid options, but the "Saint Patrick NPC" is the actual line which passes right through this street where I'm located and the app has my correct location, which I show on the next photo, but it doesn't tell me to "Hail and Ride" instead it tells me to walk to the "start Terminal" because there's the official "stop" aka end of the line.

Between there and down this whole road there's no official stop - but the "Hail and Ride" principle.

![image](https://user-images.githubusercontent.com/15773876/162641652-ae559f5a-71d8-4b0b-b28f-f3c0b06ea596.png)
This photo shows what I mean. It could tell the user "wait" and use the "hail and ride" you don't need to walk - just wait for the jeep ..

So again, the 7 options OSMAnd is showing to walk to an official stop is valid!!! No need to remove that functionality or change it - what I'm asking for is the additional possibility to show to the user that he can just wait for the "hail and ride" and doesn't need to go anywhere!!! Which is very practical to avoid the slippery slopes, steps, roads especially when it's raining.

The following link is on the openstreetmap.org the location where the examples I made are located:
[https://www.openstreetmap.org/#map=18/16.41258/120.56381&layers=ON](url)

The following photo shows the selection in JOSM:
![image](https://user-images.githubusercontent.com/15773876/162641741-7350182d-5275-4c92-b16f-e21e6129096c.png)

The following shows the Tags/Memberships in JOSM for that road:
![image](https://user-images.githubusercontent.com/15773876/162641750-80f9304d-7e0f-4550-b268-39e9b71cb0a1.png)

And the following is what it looks like in the "Edit Relition" window in JOSM:
![image](https://user-images.githubusercontent.com/15773876/162641826-f8e3f79e-4e3e-4ad0-bbb9-6a4fb5bd5156.png)

I really wish to point out here - this is legit tagging in OSM it's approved and adopted and quite common in quite some countries the "hail and ride" concept and OSMAnd is totally missing it.

So I really wish this to be adopted ASAP so I can show this to people to promote OSM here .

Thank you for your consideration.

**Comment by tradavyvy** [comment]
👍👍👍

**Comment by contrapunctus-1** [comment]
Duplicate of #6696 and #13481 (the latter also created by @duiveldoder - could you please just add comments to existing issues instead of creating new ones?)

**Comment by vshcherb** [comment]
Merged to #6696

**Comment by duiveldoder** [comment]
Thanks viktor. I sadly didn't remember anymore I already made a request.

Also I didn't know there's already an old request from many years ago - never looked for it honestly as I suspected such a very important feature in many parts of the world most notably Asia/Africa/Latin Americas would have been implemented already if it were years ago reported already.

So my excuses for the ignorance on my end.
